wondrous-app/pages/organization/[username]/members.tsx

```diff
@@ -49,7 +49,7 @@ function OrgMemberPage() {
       }}
     >
       <EntitySidebar>
-        <MemberRequests orgData={org} />
+        <Members orgData={org} />
```

Comment:
there's no need to pass in this prop. we can get the orgData information from the context